api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -26,4 +34,83 @@
 public interface Inventory extends AutoCloseable {
 
     Tenants.ReadWrite tenants();
+
+    /**
+     * Quickly navigate to the provided tenant.
+     *
+     * @param tenant the tenant to steer to.
+     * @return the access interface to the tenant
+     * @throws EntityNotFoundException if the tenant is not found
+     */
+    default Tenants.Single steerTo(Tenant tenant) throws EntityNotFoundException {
```

Comment:
Helper class with static methods could be perhaps better here. To me `inventory.steerTo(..)` evokes that I somehow changed the internal state of the `inventory` instance, but that's not the case, it only "prepares" the query and returns it as a Single or Multiple object, waiting for the entity/entities call (or other calls).

What about "look up" or "find"
`Find.in(inv).environment(env)`
because neither looking up, nor finding can change the state. But again, here it's not clear that the result wont be the environment, but something that needs to be materialized.

I am perhaps over-complicating things, here.